docs/test/button/ds4/index.html

```diff
@@ -8,6 +8,7 @@
         <ol>
             <li><a href="button-text.html">Text Button</a><iframe src="button-text.html" tabindex="-1" tabindex="-1"></iframe></li>
             <li><a href="button-icon.html">Text + Icon</a><iframe src="button-icon.html" tabindex="-1" tabindex="-1"></iframe></li>
+            <li><a href="button-wrapping.html">Text wrapping</a><iframe src="button-wrapping.html" tabindex="-1" tabindex="-1"></iframe></li>
```

Comment:
I loaded the ds4 docs on locahost, but I could not see the doc for the button wrapping.

Reply:
Good catch. Let me move it to the common examples.

Reply:
@agliga Oh, this is a test file. Were you looking for it to be added to documentation?